app/views/admin/shared/_sidenav_users.html.haml

```diff
@@ -1,7 +1,11 @@
 %ul.sidebar-nav
   - if current_ability.can?(:administer, User)
     = tab t("activerecord.models.user", count: 2), facility_users_path, (sidenav_tab == "users")
-  - if current_ability.can?(:send_bulk_emails, current_facility)
-    = tab t("bulk_email.title"), facility_bulk_email_path, (sidenav_tab == "bulk_email")
+
+  = render_view_hook "after_facility_users",
+    ability: current_ability,
```

Comment:
Won't the included partial have access to `current_ability` and `current_facility` helpers? Do we need to pass them in?